Scripts/Proxies/DefaultProxyRay.cs

```diff
@@ -139,24 +91,66 @@ public void SetLength(float length)
 
 		public void SetColor(Color c)
 		{
+			m_LineRenderer.SetColors(c, c);
```

Comment:
Hm. I believe this is how it used to work, as well. That could also explain why the line renderer was giving us that non-clamped alpha issue. I will see if I can safely remove this.